contracts/stake/src/state.rs

```diff
@@ -230,6 +261,14 @@ impl StakeState {
 
         // Update the total slashed amount
         self.slashed_amount += to_slash;
+
+        rusk_abi::emit(
+            "slash",
```

Comment:
Additionally to this, you should add an events called "shifted" at line 259, and report the eligibility as value of the event